mobile/src/views/RegistrationView.jsx

```diff
@@ -44,6 +44,7 @@ const RegistrationView = () => {
             return result.json();
           })
           .then(data => {
+            data["verification_image"] == ""  || data["verification_image"] == undefined ? data["verification_image"] = "" : null;
```

Comment:
Same comment as before

Reply:
I had to keep this logic because of the way the backend was kinda set up. If I change it up now, I gotta change a bunch of the other logic too. Can I add comments to tell people what I'm doing?